app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::SheltersController < ApplicationController
+  def index
+    @shelters = Shelter.all.reverse_alphabetical
```

Comment:
You don't want to call `.all` and `.reverse_alphabetical`. You will just use `.reverse_alphabetical` which is grabbing all of them and sorting them.